packages/core/src/global/_variables.scss

```diff
@@ -8,7 +8,7 @@ $ray-font-stack-sans-serif: 'Apercu', Avenir, -apple-system, BlinkMacSystemFont,
 $ray-font-stack-mono: 'Apercu Mono', SFMono-Regular, Menlo, Monaco, Consolas,
   'Liberation Mono', 'Courier New', monospace;
 $ray-base-font-size: rem(16px);
-$ray-base-line-height: 1.75rem;
+$ray-base-line-height: 1.618;
```

Comment:
Why change the global line height?

Reply:
it's necessary

Reply:
The global line height should be relative to the element not the base font-size

Reply:
So this is an unrelated fix?

Reply:
it's somewhat related